### DISCUSSION TITLE
add git commit diagramms

### DISCUSSION THREAD

**Discussion by evantill** [plantuml/plantuml#1342] [Ideas]
examples : https://github.com/nicoespeon/gitgraph.js

other example found on the net (from diagrams net)

<img width="1676" alt="Flowchart_Maker___Online_Diagram_Software" src="https://user-images.githubusercontent.com/609288/226146765-71092e1e-f5f6-433e-bce3-d87bd7561c2f.png">

**Comment by The-Lum** [comment]
Hi @evantill,

Here are some elements:
- https://forum.plantuml.net/5184/generate-git-graph
- https://forum.plantuml.net/5184/generate-git-graph?show=15627#c15627

Regards.
